.buildkite/Dockerfile.gpu

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:11.2.0-cudnn8-devel-ubuntu20.04
+FROM nvidia/cuda:11.2.0-cudnn8-devel-ubuntu18.04
```

Comment:
Please don't do that - we need to use the same Ubuntu in all our build jobs, otherwise we will have to maintain different dependency install mechanisms. This bit us in the past, so let's stay with a single Ubuntu version

Reply:
Ok changed back!